src/app/map/[id]/Tile/item.buttons.static.tsx

```diff
@@ -1,100 +1,138 @@
+"use client";
```

Comment:
Same thing as earlier but next's Link is already used here. Remove useSearchParams and drill down the new URLInfo prop here instead. This component then should not need to be a client component.